CHANGELOG.md

```diff
@@ -155,7 +155,7 @@ MobX 5 is the first MobX version fully leveraging Proxies. This has two big adva
 
 ### The system requirements to run MobX has been upped
 
-* MobX 5 can only be used on environments that support `Proxies`. In practice this means, no Internet Explorer (Edge is fine). No nodejs < 6. React Native on Android only when JavaScript core is [upgraded](https://github.com/react-community/jsc-android-buildscripts#how-to-use-it-with-my-react-native-app). All modern browsers are supported.
+* MobX 5 can only be used on environments that support `Proxies`. In practice this means, no Internet Explorer (Edge is fine). No nodejs < 6. React Native supported from iOS 10, on Android from RN 0.59 (or with manual JavaScript core [upgrade](https://github.com/react-community/jsc-android-buildscripts#how-to-use-it-with-my-react-native-app)), new [Hermes runtime](https://github.com/facebook/hermes/issues/28) is not supported. All modern browsers are supported.
```

Comment:
I think it might be better to separate out all RN related information to an additional bullet point. Same for the update in readme.

Reply:
Is
```
In practice this means:
- no Internet Explorer (Edge is fine)
- nodejs >= 6
React Native:
- iOS >= 10
- Android from RN 0.59 (or with manual JavaScript core [upgrade](https://github.com/react-community/jsc-android-buildscripts#how-to-use-it-with-my-react-native-app))
- new Hermes runtime is [not supported](https://github.com/facebook/hermes/issues/28)
```
will be ok?

Reply:
Sure, sounds reasonable :) I would drop a word `new` as it's not relevant and might be one day "old" :)